### ISSUE TITLE
[Bug Report] v-select accessibility aria-labelledby doesn't work

### ISSUE THREAD

**Issue by chrmorqmatic** [vuetifyjs/vuetify#9680]
### Environment
**Vuetify Version:** 2.1.10
**Vue Version:** 2.6.10
**Browsers:** Chrome 78.0.3904.97
**OS:** Windows 10

### Steps to reproduce
use the keyboard to navigate the v-select, when in opened menu use arrow to move from one element to the other

### Expected Behavior
Screen readers should say the current element linked with aria-labeledby

### Actual Behavior
The screen readers says nothing or 'blank' instead of the text
tested on
Chromevox extension: says nothing
NVDA : https://www.nvaccess.org/ says 'blank'

### Reproduction Link
<a href="https://codepen.io/chrmorqmatic/pen/XWWPyoB" target="_blank">https://codepen.io/chrmorqmatic/pen/XWWPyoB</a>

### Other comments
Tested also with official doc on https://vuetifyjs.com/en/customization/accessibility with slots or without, same behavior, same on v-select exemples

**Comment by johnleider** [comment]
According to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute we are providing the aria label definition a link to its text:

![image](https://user-images.githubusercontent.com/9064066/72281021-40e67880-35ff-11ea-85da-4cbe5dbc58a5.png)

Will need some additional help with resolving this otherwise.

**Comment by rkrasnan** [comment]
According to the persons at NVDA, this issue is caused by id duplication.

**Comment by johnleider** [comment]
@rkrasnan can you check if #10226 works properly with NVDA? Thank you

**Comment by phiter** [comment]
Since we use an activator element which retains the focus during the navigation between the options, `aria-labbeledby` will not work.

We need to use `aria-activedescendant` in the activator, passing the ID of the option element.

This is being worked on by @KaelWD in the PR mentioned by @johnleider in the comment above.
